nordeus/crowd-user-bundle/2.0/config/packages/nordeus_crowd_user.yaml

```diff
@@ -0,0 +1,16 @@
+nordeus_crowd_user:
+    # REQUIRED:
```

Comment:
Lets replace this comment with a link to the docs.

Reply:
Will do.